processing/src/test/java/io/druid/query/timeseries/TimeseriesQueryRunnerTest.java

```diff
@@ -2562,7 +2562,31 @@ public void testTimeSeriesWithSelectionFilterLookupExtractionFn()
         optimizedRunner.run(query, CONTEXT),
         Lists.<Result<TimeseriesResultValue>>newArrayList()
     );
-    TestHelper.assertExpectedResults(expectedResults, results2);
+    // post aggs are calculated in FinalizeResultsQueryRunner.run()
+    // toolChest.mergeResults doesn't do post aggs, so addRowsIndexConstant is not included in expectedResults2
+    List<Result<TimeseriesResultValue>> expectedResults2 = Arrays.asList(
```

Comment:
Done. @gianm 

Previously, I tried to wrap it with FinalizeResultsQueryRunner, but GroupByTimeseriesQueryRunnerTest failed with cast exception which is weird, then I chose to skip post aggs from the expected results.

Now I finally found the reason why GroupByTimeseriesQueryRunnerTest failed: it should pass the `finalize` context from timeSeries query to groupby query